### ISSUE TITLE
Prompt to generate paper key in chat client leads to dead-end 

### ISSUE THREAD

**Issue by socketbox** [keybase/keybase-issues#3243]
There's an orange dot by a smiley face icon. I click it.

Oh, it's not a message, just a reminder to generate a paper key. I click on an elliptical button that reads "Create a paper key."

I'm taken to a list of my devices. Odd. Well, I guess I'll click one...

Shown a device and its details, along with a red button that is not germane to paper key generation.

Where do I go from there?

MacOS 10.12.6
Keybase 1.0.48-20180516160420+b56432f520

**Comment by socketbox** [comment]
A friend pointed out that I'm supposed to click on the plus-sign to "Add new..." I'm not going to elaborate upon why this is unintuitive.

Perhaps keep this open as a UI enhancement request?

**Comment by malgorithms** [comment]
yeah, this is a known UX annoyance, and we can leave this open. In a release sometime soon that experience will be better.
